pagopa/io-app
# chore: [IOBP-492] BonusVacanze removal

## Short description
Removed BonusVacanze

## List of changes proposed in this pull request
- removed screens and components
- removed logic where possible
- removed I18n keys (with the `remove_unused_locales` script, so some other unused keys were removed as well 😉 )

## How to test
1. open `dev-server`
2. `yarn generate`
3.  run the app
4. make sure that nothing breaks on launch
5. make sure that all vaguely "bonus-related" flows run correctly. this includes, but is not limited to:
- CGN
- wallet home
- new bonus onboarding
- new payment method onboarding
- possible old transaction history
6. run `yarn jest` to make sure it all runs smoothly.
